src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -2296,11 +2242,11 @@ TerraCore:hasHostOrganism
   rdfs:domain TerraCore:Antibody ;
   rdfs:label "has host organism" ;
   rdfs:range [
-    owl:unionOf (
-      obo:OBI_0100026 
-      xsd:string 
-    );
-  ];
+      owl:unionOf (
+          obo:OBI_0100026
```

Comment:
Can't unionOf object and datatype

Reply:
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)